Plugins/IngameDebugConsole/Scripts/InputHelper.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using UnityEngine;
+
+#if ENABLE_INPUT_SYSTEM
+using UnityEngine.InputSystem;
+
+namespace IngameDebugConsole
+{
+    public static class InputHelper
+    {
+        public static bool GetKeyDown(KeyCode key)
+        {
+            bool ok = Enum.TryParse(key.ToString(), out Key newKey);
+            return ok && Keyboard.current[newKey].wasPressedThisFrame;
+        }
+    }
+}
+#elif ENABLE_LEGACY_INPUT_MANAGER
```

Comment:
Instead of #elif, we must consider the case where both the legacy and the new Input systems are enabled.

Reply:
When you active the new Input System, Unity seems to block the usages of the old one. Do you have an example where the two ones are enabled together ?

Reply:
In Player Settings, there is (or was, I don't know the current state) an option to enable both systems simultaneously.

Reply:
Ok, now it's better i think.
If NEW activated => first case, else => second case